test/dummies/dummyFile.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Create a dummy file, with overridden value of given param
+ *
+ * @param file
+ * @returns {*&{path: string, name: string, icon: string, id: string, _id: string, dir_id: string, type: string}}  a dummy file
```

Comment:
What kind of jsdoc this is?

Reply:
this is generated by my IDE.

* means any type. (because i didn't set the type of the file)

& means intersection

I will reformulate.

Reply:
I don't see anything related to this syntax on https://jsdoc.app/ . Can you point me the doc please?

Reply:
I don't know exactly this syntax. My IDE generates it.

About: Allows any type
```
/**
 * @param {*} somebody - Whatever you want.
 */
```

I guess about intersection:
https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types